feat(kibbeh): add speaker badge near to username in chat

junedkhatri31 opened the pull request:
fix #1413

**THIS IS INPROGRESS**

Need to performance test once with at least hundreds of users and thousands of messages.

junedkhatri31 commented:
Can anyone help me what would be the best way to performance test this?
I am concerned about this code
```js
const user = users.find((u) => u.id === m.userId);
```

junedkhatri31 commented:
<img width="333" alt="Screenshot 2021-05-02 at 2 07 30 PM" src="https://user-images.githubusercontent.com/20799038/116807943-ae4f4d00-ab53-11eb-83ab-8f0fcead3d82.png">
This is how it looks like

ajmnz commented:
> <img alt="Screenshot 2021-05-02 at 2 07 30 PM" width="333" src="https://user-images.githubusercontent.com/20799038/116807943-ae4f4d00-ab53-11eb-83ab-8f0fcead3d82.png">
> 
> This is how it looks like

Add the emoji before the username and use twemoji instead of the Apple ones. (maybe @goldyydev can point you in the right direction on that)

junedkhatri31 commented:
Updated.
<img width="335" alt="Screenshot 2021-05-02 at 5 31 21 PM" src="https://user-images.githubusercontent.com/20799038/116812972-1cedd400-ab6f-11eb-9d8c-1f11fb14a879.png">

Changed css in global bcoz emoji was reflecting display block which consume width 100%. `emoji` css is only used in Settings label. I've checked, nothing breaks there.

benawad commented on kibbeh/src/modules/room/chat/RoomChatList.tsx:
```diff
@@ -43,6 +45,16 @@ export const RoomChatList: React.FC<ChatListProps> = ({ room }) => {
+    if (isSpeaker) {
+      emoji = "📣";
+    }
+    return emoji && <Twemoji text={emoji} style={{ marginRight: "1ch" }}/>;
```
can you add a title to it so people can see what it means on hover

junedkhatri31 replied:
Done
